_alerts/2015-07-04-spv-mining.html

```diff
@@ -9,7 +9,7 @@
 update: 2015-07-06 02:00.  All times are UTC.</em></p>
 
 **Note: this alert is on-going: the situation has not yet been
-resolved. ([Update #1](#update-1))**
+resolved. ([Read more](#cause))**
```

Comment:
Here I kind of wanted to provide as easy way for people who had read previous versions of the alert to get straight to the new information.